allennlp/tools/drop_eval.py

```diff
@@ -242,3 +250,6 @@ def evaluate_prediction_file(prediction_path: str, gold_path: str) -> Tuple[floa
                         help='location of the prediction file')
     args = parser.parse_args()
     evaluate_prediction_file(args.prediction_path, args.gold_path)
+
```

Comment:
This, and other issues, are making pylint complain.  You can check the pylint issues locally by running `./scripts/pylint.sh`.  Be sure that your editor uses unix line endings.